README.md

```diff
@@ -23,6 +23,11 @@ docker run -it --rm -p 8081:8081 goalert/demo
 
 GoAlert will be running at [localhost:8081](http://localhost:8081). You can log in with `admin`/`admin123`.
 
+If you're using the demo container for integration testing:
+- A non-admin user is available as `user`/`user1234`.
+- You can specify the ENV variable `SKIP_SEED=1` to skip the initial seed data step.
+- You can get a session token via `curl -XPOST -H 'Referer: http://localhost:8081' -d 'username=admin&password=admin123' http://localhost:8081/api/v2/identity/providers/basic?noRedirect=1`.
```

Comment:
```suggestion
- You can get a session token via `curl -XPOST -H 'Referer: http://localhost:8081' -d 'username=admin&password=admin123' 'http://localhost:8081/api/v2/identity/providers/basic?noRedirect=1'`.
```

Reply:
Might want to add single quotes around this so the url params do n't get interpreted as variables or commands (depending on the shell where it is executed).